docs/release-notes.md

```diff
@@ -1,3 +1,27 @@
+## 0.16.0
+
+July 16, 2021
+
+### Added
+ * Added [Encode](https://github.com/sponsors/encode) funding option
+   [#1219](https://github.com/encode/starlette/pull/1219)
+
+### Fixed
+ * `starlette.websockets.WebSocket` instances are now hashable and compare by identity
+    [#1039](https://github.com/encode/starlette/pull/1039)
+ * A number of fixes related to running task groups in lifespan
+   [#1213](https://github.com/encode/starlette/pull/1213),
+   [#1277](https://github.com/encode/starlette/pull/1277)
+
+### Deprecated/removed
+ * The method `starlette.templates.Jinja2Templates.get_env` was removed
```

Comment:
Please add a link to the PR as people may be concerned about this change.

Reply:
```suggestion
 * The method `starlette.templates.Jinja2Templates.get_env` was removed
  [#1218](https://github.com/encode/starlette/pull/1218)
```